web_poet/fields.py

```diff
@@ -55,6 +62,9 @@ def field(method=None, *, cached: bool = False, meta: Optional[dict] = None):
     The ``meta`` parameter allows to store arbitrary information for the field,
     e.g. ``@field(meta={"expensive": True})``. This information can be later
     retrieved for all fields using the :func:`get_fields_dict` function.
+
+    The ``out`` parameter is an optional list of field processors, which are
```

Comment:
I wonder if it'd be useful to add `out` to FieldInfo

Reply:
done